src/Components/NavBar.js

```diff
@@ -0,0 +1,34 @@
+import Container from "react-bootstrap/Container";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+import NavDropdown from "react-bootstrap/NavDropdown";
+
+function BasicExample(props) {
```

Comment:
File is called NavBar, component called BasicExample. The component should reflect the file name

Reply:
If it is due to the import of `Navbar`, you can rename that Navbar import to possibly BootstrapNavbar or something